integrations/intercom.mdx

```diff
@@ -5,18 +5,45 @@ description: "Ingest customer conversations and support tickets to Monterey for
 
 ![intercom](/images/inter.png)
 
-### Set up
+### Overview
 
-You can find Intercom integration on your **[integration setting page](https://app.monterey.ai/w/default/settings/integrations/)**. Follow the instructions to connect your Intercom account. Anyone who has access to both your Monterey workspace and Intercom team can set up Intercom integration.
+The integration with Intercom is a great way to get product insights with and from your support team. With the integration, your team can:
 
-### Create feedback from customer conversations
+- Surface trends from customer conversations
+- Extract common feature requests, bug reports, and questions
+- Understand what prospective customers are asking about
+
+### Get Started
+
+You can find the Intercom integration on your **[integration settings page](https://app.monterey.ai/w/default/settings/integrations/intercom)**.
+
+A user with access to both Monterey AI as well as Intercom will need to authorize the connection.
+
+1. Go to "Settings"
+2. Click "Integrations" in the navigation menu
+3. Click "Intercom"
+4. Click "Connect"
+5. If necessary, log in to Gong using the appropriate account
+6. Review and approve the permissions requested
+7. Once confirmed, you will be returned to Monterey AI, where you can toggle on conversation ingestion
+
+### Intercom Inbox App
 
 Once the setup is done, you will see a Monterey AI section in the Intercom details bar on the right when you open a conversation. It will bring up a form which requires a description. You can also optionally include a customer emails and customer names from the Intercom conversation, select a project. You can then create feedback directly Intercom by clicking “Create feedback”. The feedback will show up in “All feedback” tab in Monterey auto categorized.
 
-### Automatically ingest Intercom conversations
+Anybody with access to your Intercom inbox will be able to send feedback to Monterey AI, via the app in the sidebar, even if they do not have access to Monterey AI. Only teammates added to Monterey AI directly will be able to view feedback and insights.
+
+### Ingest Conversations
+
+Once Monterey AI is authorized to access Intercom on your behalf, you can enable a setting on the **[integration settings page](https://app.monterey.ai/w/default/settings/integrations/intercom)** to have Monterey ingest new conversation data regularly. This process will run every multiple times a day, up to hourly, depending on your subscription plan.
```

Comment:
> depending on your subscription plan.

do we actually have logic for this somewhere? Just curious!

Reply:
Right now, no! But, I think from what we've seen, this could be a kind of upsell / enterprise feature we offer (e.g. we'll update your data more frequently), so kind of a nudge to see if anyone bites on it!